### PR TITLE
Updated images to UI.2

### PR DESCRIPTION
**Pull request by acron0**

### CODE REVIEW

**Comment by mrchrisadams** [comment]
Looks good - but what's the dragon all about?

**Comment by acron0** [comment]
It's Anglo-Saxon imagery. Replaced with a [knot](https://github.com/MastodonC/witan.ui/blob/d55837a095143c2608c45457bfad9beaba9ae0b8/resources/public/img/corner3.svg) now, actually. But I'm not updating the docs again, just for the icon :p

File: GUIDE.md [hunk 1]
```diff
@@ -62,6 +62,10 @@ Clicking on the blue button will display the Data selection form for that catego
 
 From here there are two options; either select an existing Data item from the items listed (and confirm with the 'Use' button), or *upload* a new Data item. It's also possible to download an existing Data item, for use as a template or reference, by selecting it and then using the 'Download' button.
 
+![Projection - Uploading](imgs/upload00.png) 
+
+To upload a file, use the 'Choose File' button and select the desired data source. There is then an option to eithre give this data source a brand new name, or to use in an existing name in which case the version number of the existing data source will be incremented. Clicking 'Upload' will begin the process.
```
Comment (mrchrisadams) [inline GUIDE.md]: small typo: `either`

Comment (mrchrisadams) [inline GUIDE.md]: Maybe break this into two sentences ? This sentence is nearly 60 words long, and I had trouble following all of it the first time. Breaking it into two smaller ones is likely to make it easier to comprehend.

I'm basing this recommendation on this post here, about keeping sentences short, ideally 25 words or less.

https://insidegovuk.blog.gov.uk/2014/08/04/sentence-length-why-25-words-is-our-limit

Comment (acron0) [inline GUIDE.md]: I only count 36 words, but I will try to word it better.

**Comment by mrchrisadams** [comment]
Looks good to me. We make the change live when we make the rest of the UI available, right?

**Comment by acron0** [comment]
Yep, I won't merge until a) you're happy and b) the UI is live

**Pull request merged**